docs/interface.md

```diff
@@ -7,8 +7,8 @@ All interactions with the Exercism website are handled automatically. Test runne
 - A test runner should provide an executable script. You can find more information in the [docker.md](./docker.md) file.
 - The script will receive three parameters:
   - The slug of the exercise (e.g. `two-fer`).
-  - A path to an input directory containing the submitted solution file(s) and the necessary test file(s) (with a trailing slash).
-  - A path to an output directory (with a trailing slash).
+  - A path to an input directory containing the submitted solution file(s) merged with the exercise file(s) from the repository (with a trailing slash). This directory should be considered as read-only. As [@ccare specified](https://github.com/exercism/automated-tests/issues/38#issuecomment-576335584) it's technically possible to write into it but it's better to use `/tmp` for temporary files (e.g. for compiling sources).
```

Comment:
Nice. The only thing I'm not sure of is "the exercise file(s) from the repository." I'm not sure people will understand. Maybe something like:

```suggestion
  - A path to an input directory containing the submitted solution file(s) and any other exercise file(s) (with a trailing slash). This directory should be considered as read-only. As [@ccare specified](https://github.com/exercism/automated-tests/issues/38#issuecomment-576335584) it's technically possible to write into it but it's better to use `/tmp` for temporary files (e.g. for compiling sources).
```

Reply:
I've updated @ErikSchierboom's suggestion to move the `(with a trailing slash)` to earlier in the sentence.

Reply:
Sounds good.